schema/schemaspec/spec.go

```diff
@@ -67,8 +67,8 @@ type (
 
 	// TypeSpec represents a specification for defining a Type.
 	TypeSpec struct {
-		Name       string
-		T          string
+		Name       string // Name is the identifier for the type in an Atlas DDL document.
+		T          string // T is the database identifier for the type.
```

Comment:
```suggestion
		// Name is the identifier for the type in an Atlas DDL document.
		Name       string
		 // T is the database identifier for the type.
		T          string
```
Comments are too long IMO for inline.